index.html

```diff
@@ -4,42 +4,40 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/styles.css" />
     <title>Document</title>
 </head>
 <body>
     <header class="header">
         <h1>Quiz App</h1>
     </header>
         <main>
-            <section>
+            <section class="section">
                 <p>
                     What header flips the axes in flexbox?
-                    <br>
-                    <button>
-                        Show Answer
-                    </button>
-                </p>
-                <p>
-                    #html
-                    #flexbox
-                    #css
                 </p>
+                <button>
+                    Show Answer
+                </button>
+                <div class="tags">
+                    <p>#html</p>
+                    <p>#flexbox</p>
+                    <p>#css</p>
+                </div>
 
             </section>
-            <section>
+            <section class="section">
                 <p>
                     Lorem Ipsum dolor sit?
-                    <br>
-                    <button>
-                        Show Answer
-                    </button>
                 </p>
-                <p>
-                    #html
-                    #flexbox
-                    #css
-                </p>
-
+                <button>
+                    Show Answer
+                </button>
+                <div class="tags">
+                    <p>#html</p>
+                    <p>#flexbox</p>
+                    <p>#css</p>
+                </div>
```

Comment:
you could also do it as a list
```suggestion
                <ul class="tags">
                    <li>#html</li>
                    <li>#flexbox</li>
                    <li>#css</li>
                </ul>
```